### PR TITLE
Add repeat-scope parameter

### PR DESCRIPTION
**Pull request by gdyuldin**
This parameter allows to change test execution order.
For example, with `--repeat-scope session` all tests executes
first time, next all tests executes second time and etc.

### CODE REVIEW

**Comment by gdyuldin** [comment]
Fixes #19

**Comment by gdyuldin** [comment]
Applying `@pytest.mark.repeat(n)` to the test class works like applying this mark to each test in this class.
README updated.

File: README.rst [hunk 1]
```diff
@@ -60,6 +60,14 @@ can use the :code:`@pytest.mark.repeat(count)` decorator:
    def test_repeat_decorator():
        pass
 
+If you want to override default tests executions order, you can use :code:`--repeat-scope`
+command line option with one of the next values: :code:`session`,  :code:`module`, :code:`class` of :code:`function`(default).
```
Comment (davehunt) [inline README.rst]: Typo: 'of' instead of 'or'

Comment (davehunt) [reply README.rst]: Also, `function` is not rendoring correctly, see https://github.com/gdyuldin/pytest-repeat/blob/31c85c7f9398b729c8c4abf46f491ea6512054ea/README.rst. I suspect you need a space before "(default)".

File: README.rst [hunk 2]
```diff
@@ -60,6 +60,14 @@ can use the :code:`@pytest.mark.repeat(count)` decorator:
...
+It behaves like a scope of the py.test fixture.
```
Comment (davehunt) [inline README.rst]: Please use "pytest" and not "py.test", as the dot was dropped in version 3.

File: README.rst [hunk 3]
```diff
@@ -60,6 +60,14 @@ can use the :code:`@pytest.mark.repeat(count)` decorator:
...
+
+:code:`function` (default) scope repeats each test :code:`count` or :code:`repeat` times before executing next test.
+:code:`session` scope repeats whole tests session, i.e. all collected tests executed once, then all such tests executed again and etc.
+:code:`class` and :code: `module` behaves similar, but repeating tests block is all tests for class or module.
```
Comment (davehunt) [inline README.rst]: `module` is not formatted, likely due to the space after ":code:"

Comment (davehunt) [reply README.rst]: It's not clear what you mean by "repeating tests block is all tests for class or module". Could you rephrase?

**Comment by gdyuldin** [comment]
I've updated README
